sccm/core/plan-design/hierarchy/log-files.md

```diff
@@ -411,7 +411,7 @@ The log file SMS_DM.log on the site system server also records communication bet
 |Log name|Description|Computer with log file|  
 |--------------|-----------------|----------------------------|  
 |objreplmgr.log|Records details about the replication of software updates notification files from a parent site to child sites.|Site server|  
-|PatchDownloader.log|Records details about the process of downloading software updates from the update source to the download destination on the site server.|Computer that hosts the Configuration Manager console from which downloads are initiated|  
+|PatchDownloader.log|Records details about the process of downloading software updates from the update source to the download destination on the site server.|When downloading updates manually, this will be located in the %temp% directory of the user running the console on the machine you're running the console. For Automatic Deployment Rules, this will be located on the site server in %windir%\CCM\Logs if the ConfigMgr client is installed on the site server|  

```

Comment:
A few suggested edits for style:

```suggestion
|PatchDownloader.log|Records details about the process of downloading software updates from the update source to the download destination on the site server.|When you manually download updates, this file is in your `%temp%` directory on the computer where you use the console. For automatic deployment rules, if the Configuration Manager client is installed on the site server, this file is on the site server in `%windir%\CCM\Logs`.|  
```

FYI, if you accept my suggestion, note that our automation process may auto-close this pull request. But that's ok! I'll reopen and finalize the pull request.